confdb/aclocal_threads.m4

```diff
@@ -10,3 +10,13 @@ AC_DEFUN([PAC_ARG_THREAD_PACKAGE], [
         uti - POSIX threads plus Utility Thread Offloading library
         none - no threads
      ],,with_thread_package=posix)])
+
+dnl PAC_ARG_POSIX_MUTEX
+dnl  - Provide configure option to select alternate mutex when posix thread package used. Defaults is to use
+dnl    pthread mutex.
+AC_DEFUN([PAC_ARG_POSIX_MUTEX], [
+     AC_ARG_WITH([posix-mutex],
+     [  --with-posix-mutex=option     Posix thread mutex to use. Supported options include:
+        default - pthread mutex.
+        ticketlock - use ticketlock, a fair lock based on atomics and busyloop
+     ],,with_posix_mutex=default)])
```

Comment:
Can you change the `pthread mutex` version to `native` instead of `default`?  We might change what we consider default in the future, but that's independent of the implementations themselves.

Reply:
will do

Reply:
fixed.